actors/miner/src/lib.rs

```diff
@@ -4276,16 +4279,18 @@ fn request_update_power(rt: &impl Runtime, delta: PowerPair) -> Result<(), Actor
 fn request_terminate_deals(
     rt: &impl Runtime,
     epoch: ChainEpoch,
-    deal_ids: Vec<DealID>,
+    sectors: &BitField,
 ) -> Result<(), ActorError> {
-    const MAX_LENGTH: usize = 8192;
-    for chunk in deal_ids.chunks(MAX_LENGTH) {
+    if !sectors.is_empty() {
+        // XXX REVIEW: should we chunk this call?
```

Comment:
It is worth figuring out if we can hit internal parameter size limits and making that case impossible.  If it's already impossible I don't see a reason to chunk.  If we are worried about market call costs I think the right mechanism is to reduce the per-epoch termination processing limit.

Reply:
Looks like it's 1MiB from https://github.com/filecoin-project/ref-fvm/blob/ac2b9b499b9d6e7de4068bbbe8f26545557edb60/fvm/src/machine/mod.rs#L158

If the sector numbers were represented as a vec of 8-byte uints, this gives room for 130k of them (55 full partitions). Both cron and manual processing of early termination are currently limited by MAX_ADDRESSED_SECTORS = 25k, so I think we're good. I will add a note to that parameter though.

Reply:
@Kubuxu do we have a guarantee that in the worst case bitfield will degrade to <= 8 bytes per int?  Are there any edge cases where it can perform worse we should worry about?